src/main.cpp

```diff
@@ -25,59 +25,62 @@
 
 #include "vcf/file_structure.hpp"
 #include "vcf/validator.hpp"
+#include "vcf/report_writer.hpp"
 
+// TODO any reason why this namespace is indented 4 spaces?
```

Comment:
Yes, you're right about coherent indentation, I just didn't noticed at first. After all this has been merged a specific commit can be created just for this so it doesn't mess up with the rest of changes.